packages/gatsby-theme-carbon/src/components/ResourceCard/resource-card.scss

```diff
@@ -40,7 +40,8 @@
   margin-bottom: 0;
 }
 
-.#{$prefix}--resource-card__icon--img .gatsby-resp-image-wrapper {
+.#{$prefix}--resource-card__icon--img .gatsby-resp-image-wrapper,
```

Comment:
Trying to avoid targeting the gatsby image wrapper. This has bitten us in the past when Gatsby changes this class or someone uses an image that is just an `img` tag wrather than markdown/gatsby-image